frontend/middleware/redirect.global.ts

```diff
@@ -36,6 +36,18 @@ export default function ({ name, params }: RouteLocationNormalizedLoaded) {
         `${v1Domain}/address/${params.id || params.slug?.[1]}`,
         { external: true }
       )
+    case 'dashboard':
+    case 'dashboard-id':
+      if (query.validators && typeof query.validators === 'string') {
+        const list = query.validators.split(',').filter((v) => {
+          return !isNaN(parseInt(v)) || isPublicKey(v)
```

Comment:
`!isNaN(parseInt(v))` is permissive. A chain like _1234abc_ passes the test.
`!isNaN(Number(v)) && String(Number(v)) === v` would be uncheatable I think.

Reply:
fixed and added a isInt() utils function